scalapb/scalapb_2.13/src/main/scala/com/linecorp/armeria/server/scalapb/ScalaPbResponseConverterFunctionProvider.scala

```diff
@@ -17,15 +17,15 @@
 package com.linecorp.armeria.server.scalapb
 
 import com.linecorp.armeria.common.annotation.{Nullable, UnstableApi}
-import com.linecorp.armeria.server.annotation.{ResponseConverterFunction, ResponseConverterFunctionProvider}
+import com.linecorp.armeria.server.annotation.{ResponseConverterFunction, DelegatingResponseConverterFunctionProvider}
```

Comment:
We could probably do
`import com.linecorp.armeria.server.annotation._` to fix the lint failure. 😉